crypto/nrf_cc310_platform/CHANGELOG.rst

```diff
@@ -9,6 +9,67 @@ Changelog - nrf_cc3xx_platform
 
 All notable changes to this project are documented in this file.
 
+nrf_cc3xx_platform - 0.9.16
+***************************
+
+New version of the library with the following features:
+
+* Added code-size optimized API for SHA-256 calculation in RAM and flash.
```

Comment:
```suggestion
* Added code-size optimized API for SHA-256.
```

Reply:
It's not clear what is in RAM/flash and it does not seem like an important detail anyway.

Reply:
It is a very important detail, as SHA-256 is not supported from FLASH in CryptoCell devices that doesn't have DMA acecss to flash

Reply:
Okay, then it should be explicitly stated what exactly "calculation in RAM and flash" means.

Reply:
Most calculations use a mix of RAM and flash. E.g. RAM for variables, and flash for code.

Reply:
Calculate is an established term in hash digest algorithms

Reply:
It's the "in RAM and flash" that I feel is not clear what is meant.

Reply:
Don't all calculations use flash and ram?

Reply:
Perhaps:

Added code-size optimized API for SHA-256 that supports both hashing data that is stored in RAM and data that is stored in flash.

Assuming that is what is meant by "calculation in RAM and flash"

Reply:
Fixed